msteams-platform/m365-apps/actions-in-m365-quick-start-guide.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Actions in Microsoft 365 quick start guide
+description: In this article, learn how to create Actions in Microsoft 365, function of Actions and its use cases.
+ms.date: 12/8/2023
+ms.author: mosdevdocs
+author: v-preethah
+ms.topic: Conceptual
+ms.subservice: m365apps
+---
+# Actions in Microsoft 365 quick start guide
+
+Get started with Actions in Microsoft 365 using the Actions across Microsoft 365 sample.
+
+## Prerequisites
+
+Before you get started, ensure that you install the following tools:
+
+| &nbsp; | Install | For using... |
+| --- | --- | --- |
+| &nbsp; | [Teams Toolkit](../toolkit/install-Teams-Toolkit.md) | A Microsoft Visual Studio Code extension that creates a project scaffolding for your app. Use the latest version. |
+| &nbsp; | [Microsoft Teams](https://www.microsoft.com/microsoft-teams/download-app) | Microsoft Teams to collaborate with everyone you work with through apps for chats, meetings, and calls in one place.|
+| &nbsp; | [Node.js](https://nodejs.org/en/download/) | Back-end JavaScript runtime environment. For more information, see [Node.js version compatibility table for project type](~/toolkit/build-environments.md#nodejs-version-compatibility-table-for-project-type).|
+| &nbsp; | [Microsoft Edge](https://www.microsoft.com/edge) (recommended) or [Google Chrome](https://www.google.com/chrome/) | A browser with developer tools. |
+| &nbsp; | [Visual Studio Code](https://code.visualstudio.com/download) | JavaScript, TypeScript, or SharePoint Framework (SPFx) build environments. Use the latest version. |
+
+After you've installed the tools, follow these steps:
+
+1. Create an [Azure SQL database](/azure/azure-sql/database/single-database-create-quickstart?view=azuresql&tabs=azure-portal&preserve-view=true).
+
+1. Add [IP address](/azure/azure-sql/database/firewall-configure?view=azuresql&preserve-view=true) of your computer into allowlist of firewall of Azure SQL Server.
+
+### Run the app with Azure
```

Comment:
Quick start guide should start with the instructions to run it locally without azure backend. So the steps you wrote in the "Build Actions in Microsoft 365".   Run the App with Azure is not the quick way to start.